guvnor-ala/guvnor-ala-wildfly/guvnor-ala-wildfly-provider/src/test/java/org/guvnor/ala/wildfly/executor/tests/requirement/NotWindowsRequirement.java

```diff
@@ -0,0 +1,17 @@
+package org.guvnor.ala.wildfly.executor.tests.requirement;
+
+import org.arquillian.cube.spi.requirement.Requirement;
+import org.arquillian.cube.spi.requirement.UnsatisfiedRequirementException;
+
+import java.util.Locale;
+
+public class NotWindowsRequirement implements Requirement<RequiresNotWindows> {
```

Comment:
I was wondering - did we actually check it on something else but Linux and Windows? If not, wouldn't it be safer to just make this `Requirement` be `OnlyLinuxRequirement`?